### ISSUE TITLE
Add the ability to flag errors from a specific host.

### ISSUE THREAD

**Issue by skunk-ink** [SiaFoundation/renterd#1018]
### Description

There are some errors that could be a temporary problem with the host. For instance, hosts that have migrated from said and have lost sectors. Depending on the reason, I am not necessarily wanting to add them to the block list if they have otherwise been proven to be a stable and reliable host. At the same time if this issue keeps happening then I want to be aware of it so I can add the host to the block list if needed.

My proposed solution is to add an action in the error log host menu that allows you to flag that specific error on that specific host. Then, if that host runs into the same error later on, there should be an indication on the alert that this this error has previously been flagged on the host before.

Example:
![flag-host-error](https://github.com/SiaFoundation/renterd/assets/22564783/db886e14-1e04-425d-86e8-6c6b2bad119f)

### Version

v1.0.5

### What operating system are you running (e.g. Ubuntu 22.04, macOS, Windows 11)?

Ubuntu 22.04.3 Server LTS

**Comment by ChrisSchinnerl** [comment]
I don't think we need to flag the host. Each alert has a unique ID.  So what I propose instead is keeping track of the last time a specific alert was dismissed.

So in this particular case, you might reset the lost sectors, then dismiss the alert and after a day the alert shows up and tells you "LastOccurrence: 1 day ago". This could possibly be extended with some more metadata such as the number of times it has occurred.
